pylib/setup.py

```diff
@@ -1,11 +1,31 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import sys
+
 import setuptools
 
-with open("../meta/version") as fh:
-    version = fh.read().strip()
+minimum_python_version = (3, 7)
+
+if sys.version_info < minimum_python_version:
```

Comment:
I'm not sure I understand the point of this, as we care about install-time checks, and that should already be covered by python_requires below.

Reply:
I did not see that. I Removed it.